frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx

```diff
@@ -127,7 +130,31 @@ export default function ApplyFlowTypeOfApplication() {
       </div>
       <div className="max-w-prose">
         {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
-        <p className="mb-6 italic" id="form-instructions">
+        <p className="mb-6">{t('apply:eligibility.type-of-application.page-description')}</p>
+        <div className="mb-6 space-y-4">
+          <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-self')}</h2>
+          <div>
```

Comment:
Remove this empty DIV

Reply:
The vertical space apply between <p> and <ul> as well, is there an alternative way to remove the space?

Reply:
We want to be aligned with other pages style. It's ok.